quest/util/config.py

```diff
@@ -71,7 +70,7 @@ def update_settings(config={}):
         config['BASE_DIR'] = os.path.join(os.getcwd(), config['BASE_DIR'])
     settings.update(config)
     settings.setdefault('BASE_DIR', _default_quest_dir())
-    settings.setdefault('CACHE_DIR', 'cache')
+    settings.setdefault('CACHE_DIR', '.cache')
```

Comment:
I think this may need to be: `settings.setdefault('CACHE_DIR', '.cache', 'user_cache')`

Reply:
You are correct, I missed that when checking yesterday. Thanks